scripts/python/singlepss.py

```diff
@@ -492,113 +532,118 @@ def buf_node_in(pssName, buf, args):
 			addr = argv[3]
 		else:
 			addr = "0x"
-		
-		# backend add recipient call	
+
+		# backend add recipient call
 		if not currentPss.add(nick, key, addr):
-			wOut(PSS_BUFPFX_ERROR, [bufs[pssName]], "!!!", "add contact error: " + currentPss.error()['description'])
+			wOut(PSS_BUFPFX_ERROR, [bufs[pssName]], "!!!",
+			     "add contact error: " + currentPss.error()['description'])
 			return weechat.WEECHAT_RC_ERROR
 
 		# refresh the plugin memory map version of the recipient
-		wOut(PSS_BUFPFX_DEBUG, [], "!!!", "added key " + key + " to nick " + nick)
+		wOut(PSS_BUFPFX_DEBUG, [], "!!!",
+		     "added key " + key + " to nick " + nick)
 		nicks[key] = currentPss.get_contact(nick)
 		remotekeys[nick] = key[2:]
 
 		# append recipient to file for reinstating across sessions
-		storeFile.write(nick + "\t" + key + "\t" + addr + "\t" + currentPss.key + "\n")
+		storeFile.write(nick + "\t" + key + "\t" + addr + "\t" +
+		                currentPss.key + "\n")
 
 		# open the buffer if it doesn't exist
-		buf_get(pssName, "chat", nick, True)	
-
-		wOut(PSS_BUFPFX_INFO, [bufs[pssName]], "!!!", "added contact '" + nicks[key].nick + "' to '" + pssName + "' (key: " + pss.label(key) + ", addr: " + pss.label(addr) + ")")
+		buf_get(pssName, "chat", nick, True)
 
+		wOut(
+		    PSS_BUFPFX_INFO, [bufs[pssName]], "!!!",
+		    "added contact '" + nicks[key].nick + "' to '" + pssName +
+		    "' (key: " + pss.label(key) + ", addr: " + pss.label(addr) + ")")
 
 	# send a message to a recipient
 	elif argv[0] == "send" or argv[0] == "msg":
 
 		if argc < 3:
-			wOut(PSS_BUFPFX_ERROR, [bufs[pssName]], "!!!", "not enough arguments for send")
+			wOut(PSS_BUFPFX_ERROR, [bufs[pssName]], "!!!",
+			     "not enough arguments for send")
 			return weechat.WEECHAT_RC_ERROR
 
 		nick = argv[1]
 		msg = " ".join(argv[2:])
-	
-		# \todo handle hex address only	
+
+		# \todo handle hex address only
 		if not currentPss.have_nick(nick):
-			wOut(PSS_BUFPFX_ERROR, [bufs[pssName]], "!!!", "invalid nick " + nick)
+			wOut(PSS_BUFPFX_ERROR, [bufs[pssName]], "!!!",
+			     "invalid nick " + nick)
 			return weechat.WEECHAT_RC_ERROR
 
 		buf = buf_get(pssName, "chat", nick, True)
 		# \todo remove the bufs dict, since we can use weechat method for getting it
 		bufs[weechat.buffer_get_string(buf, "name")] = buf
 
 		if not pss.is_message(msg):
-			wOut(PSS_BUFPFX_DEBUG, [bufs[pssName]], "", "invalid message " + msg)
+			wOut(PSS_BUFPFX_DEBUG, [bufs[pssName]], "",
+			     "invalid message " + msg)
 			return weechat.WEECHAT_RC_ERROR
 
 		return buf_in(pssName, buf, msg)
 
 	# output node key
 	elif argv[0] == "key" or argv[0] == "pubkey":
-		wOut(PSS_BUFPFX_INFO, [bufs[pssName]], pssName + ".key", currentPss.key)
-
+		wOut(PSS_BUFPFX_INFO, [bufs[pssName]], pssName + ".key",
```

Comment:
while I'm reading this, I'm even not sure if splitting these into multiple lines will make them clearer. In the end this is only output to client, and the `wOut` function prefix is very distinguishable. Maybe just revert this.